workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs

```diff
@@ -27,6 +27,8 @@ internal class RequiredFieldInjector
             = "[Require] attribute found on member that is not Injectable. This member will be ignored.";
         private const string MalformedInjectable
             = "Injectable found without required attributes, this is invalid.";
+        private const string RequirableFieldDoesNotInheritRequirableBase
```

Comment:
Wondering if this can be tested in an editor test

Reply:
@jessicafalk can it be done similarly to how you tested the systems annotations?

Reply:
that should be doable with a bit of reflection magic

Reply:
We can do this in a polish phase, I'll make a ticket

Reply:
UTY-1144